### PR TITLE
Updated Russian translation for current develop branch

### PR DESCRIPTION
**Pull request by Alexell**
Verified on the release v3.3.0

### CODE REVIEW

**Comment by Alexell** [comment]
Will be recreated.

File: src/lang/ru_RU.lang.php [hunk 1]
```diff
@@ -210,74 +251,91 @@
...
+		'pushover_error_noapp' => 'Не удалось отправить тестовое уведомление: Pushover "App API token" не найден в основных настройках.',
+		'pushover_error_nokey' => 'Не удалось отправить тестовое уведомление: Pushover ключ не найден в вашем профиле.',
+		'telegram_sent' => 'Уведомление в Telegram отправлено',
+		'telegram_error' => 'Произошла ошибка при отправке уведомления в Telegram: %s',
+		'telegram_error_notoken' => 'Не удалось отправить тестовое уведомление: Telegram API token не найден в основных настройках.',
+		'telegram_error_noid' => 'Не удалось отправить тестовое уведомление: идентификатор чата не найден в вашем профиле.',
 		'log_retention_period' => 'Период хранения логов',
 		'log_retention_period_description' => 'Количество дней хранения логов уведомлений и архива аптайма серверов. Введите 0 для выключения очистки логов.',
-		'log_retention_days' => 'дней',
+		'log_retention_days' => 'days',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php:323]: Why is it changed back to English?

File: src/lang/ru_RU.lang.php [hunk 2]
```diff
@@ -210,74 +251,91 @@
...
-		'log_status_description' => 'Если лог установлен в TRUE, монитор будет логировать все события режим которых выбран в типе уведомлений.',
+		'log_status_description' => 'Если лог статусов включен, монитор будет логировать все события выбранные в типе уведомлений.',
 		'log_email' => 'Логировать уведомления отправленые по E-mail',
 		'log_sms' => 'Логировать уведомления отправленые по SMS',
 		'log_pushover' => 'Логировать Pushover уведомления',
-		'updated' => 'Параметры были успешно применены.',
-		'tab_email' => 'E-mail',
+		'log_telegram' => 'Логировать Telegram уведомления',
+		'updated' => 'Настройки успешно сохранены.',
+		'tab_email' => 'Email',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php:282]: Is 'эмаль' the correct translation?

File: src/lang/ru_RU.lang.php [hunk 3]
```diff
@@ -210,74 +251,91 @@
...
 		'tab_sms' => 'SMS',
 		'tab_pushover' => 'Pushover',
+		'tab_telegram' => 'Telegram',
 		'settings_email' => 'Настройка E-mail',
 		'settings_sms' => 'Настройка SMS',
 		'settings_pushover' => 'Настройка Pushover',
+		'settings_telegram' => 'Настройка telegram',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php:289]: Telegram with a capital letter, please.

File: src/lang/ru_RU.lang.php [hunk 4]
```diff
@@ -210,74 +251,91 @@
...
+			'Вы будете получать уведомление при каждом запуске скрипта проверки, как только сервер перейдет в статус оффлайн, даже если сервер находится в этом статусе несколько часов.',
 		'alert_type_status' => 'Изменение статуса',
 		'alert_type_offline' => 'Оффлайн',
 		'alert_type_always' => 'Всегда',
+		'alert_proxy' => 'Даже если включено, прокси никогда не используется для сервисов',
+		'alert_proxy_url' => '<b>Формат:</b> адрес:порт',
 		'log_status' => 'Лог статусов',
-		'log_status_description' => 'Если лог установлен в TRUE, монитор будет логировать все события режим которых выбран в типе уведомлений.',
+		'log_status_description' => 'Если лог статусов включен, монитор будет логировать все события выбранные в типе уведомлений.',
 		'log_email' => 'Логировать уведомления отправленые по E-mail',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php:277]: Same as the other comment about the translation of Email.

**Pull request closed without merging**